build.gradle

```diff
@@ -63,24 +78,27 @@ ext {
   compileSdkVersion = 30
   targetSdkVersion = 30
   minSdkVersion = 24
-  androidxAppCompat = "1.2.0"
-  androidxAnnotations = "1.2.0"
-  androidMaterial = "1.3.0"
-  cardView = "1.0.0"
-  googleMaps = "17.0.1"
-  constraintLayout = "2.0.4"
-  swiperefreshLayout = "1.1.0"
-  threeTenAbp = "1.3.1"
-  picasso = "2.71828"
-  ktx = "1.3.2"
-  junit = "4.13.2"
-  stetho = "1.6.0"
-  junitAndroid = "1.1.2"
-  espressoCore = "3.3.0"
-  espressoContrib = "3.1.0"
-  testRules = "1.3.0"
-  mockitoKotlin = "2.2.0"
-  mockitoAndroid = "3.9.0"
-  coroutines = "1.5.0"
-  lint = "30.0.1"
+  androidxAppCompat = "androidx.appcompat:appcompat:1.2.0"
+  androidxAnnotations = "androidx.annotation:annotation:1.2.0"
+  androidMaterial = "com.google.android.material:material:1.3.0"
+  cardView = "androidx.cardview:cardview:1.0.0"
+  googleMaps = "com.google.android.gms:play-services-maps:17.0.1"
+  constraintLayout = "androidx.constraintlayout:constraintlayout:2.0.4"
+  swiperefreshLayout = "androidx.swiperefreshlayout:swiperefreshlayout:1.1.0"
+  threeTenAbp = "com.jakewharton.threetenabp:threetenabp:1.3.1"
+  picasso = "com.squareup.picasso:picasso:2.71828"
+  ktx = "androidx.core:core-ktx:1.3.2"
+  junit = "junit:junit:4.13.2"
+  stetho = "com.facebook.stetho:stetho:1.6.0"
+  junitAndroid = "androidx.test.ext:junit:1.1.2"
+  espressoCore = "androidx.test.espresso:espresso-core:3.3.0"
+  espressoContrib = "com.android.support.test.espresso:espresso-contrib:3.1.0"
+  testRules = "androidx.test:rules:1.3.0"
+  mockitoKotlin = "com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0"
+  mockitoAndroid = "org.mockito:mockito-android:3.9.0"
+  coroutines = "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.5.0"
+  coroutinesTest = "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.5.0"
+  lint = "com.android.tools.lint:lint:30.0.2"
```

Comment:
I think dependabot recognises having a separate version + dependency definition as long as it's still in the same file (e.g. the kotlin updates worked) - maybe we can still provide version + dependency separately where we've got multiple dependencies all using the same version - just defined in the same file. otherwise i'm not sure how upgrades work - it might try and update all three dependencies on their own and therefore it'll probably fail (same for coroutines)

Reply:
yeah e.g. this worked https://github.com/Skyscanner/backpack-android/pull/668